subprojects/docs/src/docs/release/notes.md

```diff
@@ -50,6 +51,19 @@ Example:
 ADD RELEASE FEATURES BELOW
 vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv -->
 
+<a name="ear-plugin"></a>
+### Ear Plugin
+
+It is now possible to generate valid deployment descriptors for Java EE 8, Jakarta EE 9 and Jakarta EE 10
+by specifying the corresponding version in the `deploymentDescriptor` instead of having to use a custom descriptor file.
+
+```groovy
```

Comment:
❌ We use Kotlin snippets by default in release notes

Reply:
updated